src/pages/api/comments/[accident-id].ts

```diff
@@ -7,14 +8,30 @@ import { NewComment } from "../../../types";
 const handler: NextApiHandler = async (req, res) => {
   const accidentId = req.query["accident-id"] as string;
 
-  if (req.method === "POST" && accidentId) {
+  if (!accidentId) {
+    res.status(422).json({
+      error: "missing_param",
+      description: "query param accident-id is missing",
+    });
+
+    return;
+  }
```

Comment:
Not sure this param can be missing. The route is `src/pages/api/comments/[accident-id].ts` and Next should not call the handler if we hit `/api/comments//` etc.

Reply:
Done